anta/cli/check/utils.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+# pylint: disable=too-many-arguments
+# pylint: disable=cyclic-import
```

Comment:
This should be fixed what is the issue? if it is about typing here is the correct pattern: https://stackoverflow.com/questions/39740632/python-type-hinting-without-cyclic-imports 

```
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from main import Main
```